.mockery.yaml

```diff
@@ -75,6 +75,8 @@ packages:
       all: False
     interfaces:
       Core:
+      Snapshot:
+      BlockExecutionDataReader:
```

Comment:
what happens if we remove all of the config for `github.com/onflow/flow-go/module/executiondatasync/optimistic_sync` and just let it generate all of the mocks? if it's only a couple extra we may not need, I vote we do that to keep the config simple